### DISCUSSION TITLE
Advanced copy of cell values as qouted text

### DISCUSSION THREAD

**Discussion by sykyman** [dbeaver/dbeaver#22498] [Q&A]
### Description

When I make a copy of selected cells from grid by advanced copy and choose Quote cell values, the result is not qouted
![image](https://github.com/dbeaver/dbeaver/assets/28979114/86551311-bd4f-4336-97a4-ca688bd6fff1)

### DBeaver Version

Comunity Version 23.3.2.202401072024

### Operating System

Windows 10

**Comment by LonwoLonwo** [comment]
Hello @sykyman 

You need to use another setting for data quoting - the "Always quote values" setting.

![2024-01-18 11_20_31-DBeaver Ultimate 23 3 3 - _language](https://github.com/dbeaver/dbeaver/assets/45152336/10307095-1d6a-43dc-8f87-97faab6f6f82)

The first one has another condition.

**Comment by sykyman** [comment]
Oh, thank you. My misstake. I do not know, why i used this one.
